### PR TITLE
Maintenance Page HTML Code

### PR DESCRIPTION
**Pull request by TeemoCell**
This should fix the issue in #1691 . In case i used the right `Output` method.

### CODE REVIEW

**Comment by Aberdeener** [comment]
Did you test this?

**Comment by TeemoCell** [comment]
Yeah, tested it. `Height/Width` and all other inline styles are working now.

**Comment by partydragen** [comment]
@Aberdeener its a bug with the getPurified itself, discussed in discord

**Comment by TeemoCell** [comment]
I try to fix the `getPurified` then.

**Comment by TeemoCell** [comment]
Fixed

**Comment by Derkades** [comment]
Is this safe? Is getPurified used anywhere else?

**Comment by TeemoCell** [comment]
Yeah it is used in quite a few files 
image

File: core/classes/Core/Output.php [hunk 1]
```diff
@@ -53,9 +53,10 @@ public static function getPurified(?string $input, bool $escape_invalid = false)
             $purifierConfig->set('HTML.Doctype', 'XHTML 1.0 Transitional');
             $purifierConfig->set('URI.DisableExternalResources', false);
             $purifierConfig->set('URI.DisableResources', false);
-            $purifierConfig->set('HTML.Allowed', 'u,a,p,b,i,small,blockquote,span[style],span[class],p,strong,em,li,ul,ol,div[align],br,img,figure,figcaption');
-            $purifierConfig->set('CSS.AllowedProperties', ['text-align', 'display', 'float', 'color', 'background-color', 'background', 'font-size', 'font-family', 'text-decoration', 'font-weight', 'font-style', 'font-size']);
+            $purifierConfig->set('HTML.Allowed', 'u,a,p,b,i,small,blockquote,img[style|width|height|alt|src],span[style|class],div[style|align],p,strong,em,li,ul,ol,br,img,figure,figcaption');
+            $purifierConfig->set('CSS.AllowedProperties', ['text-align', 'display', 'float', 'color', 'background-color', 'background', 'font-size', 'font-family', 'text-decoration', 'font-weight', 'font-style', 'font-size', 'width']);
             $purifierConfig->set('CSS.AllowTricky', true);
+            $purifierConfig->set('CSS.MaxImgLength', null);
```
Comment (samerton) [inline core/classes/Core/Output.php:59]: Why is this change necessary? According to the HTMLPurifier docs, it opens up to imagecrash attacks (imagine someone adding a height + width of millions of pixels 😄)

Comment (TeemoCell) [reply core/classes/Core/Output.php:59]: I've tested it myself with a 50000x50000 image but we can add add a limit.

**Comment by TeemoCell** [comment]
Changed it to `1920px` in the last commit which should prevent imagecrash attacks

**Comment by Derkades** [comment]
So this is no longer necessary since the TinyMCE switch? @Aberdeener

File: core/classes/Core/Output.php (hunk 1 above)
Comment (partydragen) [reply core/classes/Core/Output.php]: > 

i still believe its a issue, i had way to big image many years ago with made the website extremly slow with being the case @samerton telling i guess

**Comment by Aberdeener** [comment]
> So this is no longer necessary since the TinyMCE switch? @Aberdeener

i believe so. closing - if this happens again we can reopen to investigate.
